From f8f6f33509870641955846320a284c3392bb4009 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Mon, 11 Jul 2022 15:59:16 +0300
Subject: [PATCH] iot-gate-imx8plus: restore default network interfaces
 enumeration

iMX8M-Plus SoC features two network interfaces:
 - fec:  ethernet@30be0000 - eth0 by default
 - eqos: ethernet@30bf0000 - eth1 by default
Initially iot-gate-imx8plus patch used reversed enumeration achieved by
the means of alias). As a result there was inconsistency between U-boot
and Linux network interface naming.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/dts/sb-iotgimx8plus.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm/dts/sb-iotgimx8plus.dtsi b/arch/arm/dts/sb-iotgimx8plus.dtsi
index 4fcc98a889..74068a8c24 100644
--- a/arch/arm/dts/sb-iotgimx8plus.dtsi
+++ b/arch/arm/dts/sb-iotgimx8plus.dtsi
@@ -4,12 +4,6 @@
  */
 
 / {
-
-	aliases {
-		ethernet0 = &eqos;
-		ethernet1 = &fec;
-	};
-
 	chosen {
 		bootargs = "console=ttymxc1,115200 earlycon=ec_imx6q,0x30890000,115200 console=tty0";
 		stdout-path = &uart2;
-- 
2.17.1

